charts/.ci/lint-config.yaml

```diff
@@ -0,0 +1,6 @@
+rules:
+  # One blank line is OK
+  empty-lines: 
+    max-start: 1
+    max-end: 1
+    max: 1
```

Comment:
missing new line

Reply:
Ironic that a lint config fails a human lint ;)